tests/baselines/reference/tupleTypes.errors.txt

```diff
@@ -9,15 +9,11 @@ tests/cases/compiler/tupleTypes.ts(18,1): error TS2322: Type '[number, string, n
     Type '3' is not assignable to type '2'.
 tests/cases/compiler/tupleTypes.ts(41,1): error TS2322: Type '[]' is not assignable to type '[number, string]'.
 tests/cases/compiler/tupleTypes.ts(47,1): error TS2322: Type '[number, string]' is not assignable to type 'number[]'.
-  Types of property 'pop' are incompatible.
-    Type '() => string | number' is not assignable to type '() => number'.
-      Type 'string | number' is not assignable to type 'number'.
-        Type 'string' is not assignable to type 'number'.
+  Type 'string | number' is not assignable to type 'number'.
+    Type 'string' is not assignable to type 'number'.
 tests/cases/compiler/tupleTypes.ts(49,1): error TS2322: Type '[number, {}]' is not assignable to type 'number[]'.
-  Types of property 'pop' are incompatible.
-    Type '() => number | {}' is not assignable to type '() => number'.
-      Type 'number | {}' is not assignable to type 'number'.
-        Type '{}' is not assignable to type 'number'.
+  Type 'number | {}' is not assignable to type 'number'.
+    Type '{}' is not assignable to type 'number'.
 tests/cases/compiler/tupleTypes.ts(50,1): error TS2322: Type '[number, number]' is not assignable to type '[number, string]'.

```

Comment:
I looked for explicit tuple-tuple relations in isRelatedTo and couldn't find it. Can you point me to it?

Reply:
Right here https://github.com/Microsoft/TypeScript/pull/26790/files#diff-c3ed224e4daa84352f7f1abcd23e8ccaR11832